x-pack/plugins/index_management/public/application/components/component_templates/lib/api.ts

```diff
@@ -95,13 +95,6 @@ export const getApi = (
     });
   }
 
-  async function getInferenceModels() {
-    return sendRequest({
-      path: `${apiBasePath}/inference/all`,
-      method: 'get',
-    });
-  }
-
```

Comment:
Moved to x-pack/plugins/index_management/public/application/services/[api.ts](https://github.com/elastic/kibana/blob/5dc6f5f88c75c44f70bf0997de9fa55f040a83af/x-pack/plugins/index_management/public/application/services/api.ts)

Figured this endpoint is not really used by component templates but is more of index mapping use case. 

No strong opinion here. please let me know if need to move back or has an alternative option.

Reply:
It has been recenlty introduced by https://github.com/elastic/kibana/pull/179575 so its only dependencies are that feature, I think its ok to move it.